client/sagas/mouse-trap-saga.js

```diff
@@ -26,6 +25,10 @@ export default function mouseTrapSaga(actions$) {
     ...Object.keys(softRedirects)
       .map(key => bindKey(key, () => push(softRedirects[key]))),
     bindKey(
+      'g n a',
+      () => hardGoTo('https://www.freecodecamp.com/about')
```

Comment:
Could this use the `aboutUrl` from `constantStrings.json`?

Reply:
I think that would be a good idea. It does look like there's somehow some duplication going on, that stuff in main.js appears to do the same as what is done here